### PR TITLE
feat: validation files, setup for bianca and config update

### PR DESCRIPTION
**Pull request by Smeds**
### This PR:

(If this is a release PR, no need to add following. Leave this part empty)
(Use the following lines to create a PR text body. Make sure to remove all non-relevant one after you're done)
(Repeat each field as many times as necessary)

Added: for new features.
Changed: for changes in existing functionality.
Deprecated: for soon-to-be removed features.
Removed: for now removed features.
Fixed: for any bug fixes.
Security: in case of vulnerabilities.

### Review and tests:
- [ ] Tests pass
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Code review
- [ ] `CHANGELOG.md` is updated
- [ ] New code is executed and covered by tests, and test approve

### CODE REVIEW

File: config/config.data.hg19.yaml [hunk 1]
```diff
@@ -0,0 +1,143 @@
+
+PROJECT_DESIGN_DATA: ""
+PROJECT_PON_DATA: ""
+PROJECT_REF_DATA: ""
+
+reference:
+  background: "/projects/wp1/nobackup/ngs/utveckling/Twist_DNA_DATA_HG38/Background/background_panel_novaseq_lund_8.tsv"
+  artifacts: "/projects/wp1/nobackup/ngs/utveckling/Twist_DNA_DATA_HG38/Artifact/artifact_panel_novaseq_lund_8.tsv"
```
Comment (jonca79) [inline config/config.data.hg19.yaml:8]: Add variable for path

File: build/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+BRANCH_OR_TAG="add-validation-ref-yaml" bash build/build_conda.sh
+
+TAG_OR_BRANCH="add-validation-ref-yaml" bash build_container.sh
+
+#docker build --build-arg TwIST_SOLID_VERSION=develop . -f docker/Twist_DNA_Solid.dockerfile --tag twist_solid:develop
+
+#singularity build twist_solid_develop.sif docker-daemon://twist_solid:develop
```
Comment (Smeds) [inline build/README.md:8]:
```suggestion
```

File: docs/references.md [hunk 3]
```diff
@@ -1,5 +1,19 @@
...
+```bash
+# make sure hydra-genetics is available
+# make sure that TMPDIR points to a location with a lot of storage, it
+# will be required to fetch reference data
+export TMPDIR=/PATH_TO_STORAGE
+hydra-genetics --debug --verbose references download -o design_and_ref_files  -v config/references/references.hg19.yaml -v config/references/design_files.hg19.yaml -v config/references/nextseq.hg19.pon.yaml
+```
+
+### Validate if data requires update
+
```
[the remark this answers is not in the document]
Comment (Smeds) [reply docs/references.md:16]: Yes! Looks like I forgot to add the content:/

[the remark this answers is not in the document]
Comment (Smeds) [reply docs/references.md:21]: I have added some instructions now @jonca79

**Review by jonca79 — APPROVED** [review]

**Pull request merged**